src/ctl/ctl.c

```diff
@@ -287,10 +271,10 @@ static void ctl_query_cleanup_real_args(const umf_ctl_node_t *n, void *real_arg,
  */
 static int ctl_exec_query_read(void *ctx, const umf_ctl_node_t *n,
                                umf_ctl_query_source_t source, void *arg,
-                               umf_ctl_index_utlist_t *indexes,
+                               size_t size, umf_ctl_index_utlist_t *indexes,
                                const char *extra_name,
                                umf_ctl_query_type_t query_type) {
-    (void)extra_name, (void)query_type;
+    (void)extra_name, (void)query_type, (void)size;
```

Comment:
now only the query_type is not used (a.k.a. - only used in debug)

Reply:
Done